src/browser/BrowserService.cpp

```diff
@@ -388,14 +388,16 @@ QList<Entry*> BrowserService::searchEntries(Database* db, const QString& hostnam
         return entries;
     }
 
-    for (Entry* entry : EntrySearcher().search(hostname, rootGroup, Qt::CaseInsensitive)) {
+    QString base = baseDomain(hostname);
+    for (Entry* entry : EntrySearcher().search(base, rootGroup, Qt::CaseInsensitive)) {
```

Comment:
Just replace "base" with "baseDomain(hostname)", no need for a temp variable with a questionable name.